pulseaudio-control.bash

```diff
@@ -300,8 +333,22 @@ function output() {
 
 
 function usage() {
-    echo "Usage: $0 ACTION"
-    echo ""
+    echo "Usage: $0 [OPTION...] ACTION"
+    echo
+    echo "Options: (defaults)"
+    echo "    --autosync, --no-autosync             whether to maintain same volume for all programs ($AUTOSYNC)"
+    echo "    --color-mute <rrggbb>                 color in which to format when muted (${MUTED_COLOR:4:-1})"
+    echo "    --notifications, --no-notifications   whether to show notifications when changing sink ($NOTIFICATIONS)"
+    echo "    --osd, --no-osd                       whether to display KDE's OSD message ($OSD)"
+    echo "    --vol-icons <icon>[,<icon>...]        icons for volume, from lower to higher ($(IFS=, ; echo "${VOLUME_ICONS[*]}"))"
+    echo "    --vol-icon-mute <icon>                icon to use when muted ($MUTED_ICON)"
+    echo "    --vol-max <int>                       maximum volume to which to allow increasing ($MAX_VOL)"
+    echo "    --vol-step <int>                      step size when inc/decrementing volume ($INC)"
+    echo "    --sink-blacklist <name>[,<name>...]   sinks to ignore when switching ()"
+    echo "    --sink-icon <icon>                    icon to use for sink ($SINK_ICON)"
+    echo "    --sink-name-from <prop>               pacmd property to use for sink name ($SINK_NICKNAME_PROP)"
+    echo "    --sink-nickname <name>:<nick>         nickname to assign to given sink name (may be given multiple times) ()"
```

Comment:
Ah I agree, and was about to do that, but does that make it sound too much like it takes an index, or some other identifier? `pacmd` does call it a 'name'. What about changing the other instead, to `--sink-nickname-from` - since it is effectively the 'nick' of the other that it gets used as.

More I think about it that makes most sense to me - set an explicit nickname, or take it from this property.

Reply:
I would make this two lines and include a comment that indicates how to obtain these nicknames with `pactl list sinks short`. That is just as useful as the previous README and clears up what the ID is, IMO. We should also rename `--sink-name-from` to `--sink-nickname-from`.